src/uptane/uptanerepository.cc

```diff
@@ -15,6 +15,7 @@
 #include "invstorage.h"
 #include "logger.h"
 #include "openssl_compat.h"
+#include "ostree.h"
```

Comment:
One of my previous tasks, was to remove ostree dependency from uptane implementation, but this change moves it back